src/main/java/seedu/address/logic/commands/TrendCommand.java

```diff
@@ -31,7 +31,9 @@ public class TrendCommand extends Command {
      * @param args The year such that the data during this year is to be analyzed.
      */
     public TrendCommand(String args) {
-        this.year = Integer.parseInt(args.trim());
+        String trimmed = args.trim();
```

Comment:
Good improvement on code quality here! The change is quite minor but makes the code a lot more readable.